README.md

```diff
@@ -28,7 +28,7 @@ The maki module exports `layouts` which is an object that can be used to organiz
 var maki = require('@mapbox/maki');
 
 files.forEach(function(fileName, j) {
-  maki.layouts.all.forEach(function(icon) {
+  maki.layouts.forEach(function(icon) {
```

Comment:
This won't work because layouts has the `all` key

Reply:
Weird. I didn't commit the change to layout. Updated.